Chapters/2-5-Chapter12.md

```diff
@@ -0,0 +1,175 @@
+
+"Two ships?" asked Casra, incredulously. "And they're so low in the water."
+
+But Kipnu spared no time and began to paddle to intercept them.
+
+They drew closer, and the front ship began its turning maneuver. A huge crowd of people was thrown off the side all at once without bindings on their arms, and they flailed about in the water struggling to float in the disturbance.
+
+"Quickly!" Casra yelled and they began to paddle against the waves, around the people, and toward the rudder of the receding ship. Kipnu saw the clamor in the water. He had prepared himself mentally to see them, to be heartless for just this moment. But then the second boat had arrived. It began keeling, pouring more people out into the water – all of them with bound hands, and it overwhelmed him.
+
+"We're close enough!" yelled Casra, "Linyu, throw the rope!"
+
+Linyu had rehearsed this part and knew exactly what to do. Thi ran to the front of the boat and tossed a rope over the rudder in one go. In an instant, Kipnu leapt off the boat and into the water. The cold bit at his limbs but he wasn't letting it bother him - these people needed his help. The distance was great - much greater than the day before, but he knew he could do it, or at least he was pretty sure.
+
+He reached a man who hadn't moved from the spot where the ship had left him. He was gasping in shock, showing signs of confusion. _The beginnings of hypothermia_, Kipnu thought.
+
+"On your back!" Kipnu yelled at the man, but he didn't respond. Kipnu latched their arms together, and began swimming backward toward the shore until a woman came upon them.
+
+She yelled between waves, "I've got him."
+
+Kipnu understood her, and handed him off. His feet were starting to go numb, but he ignored them and kept looking for stragglers. Against the Ara horizon, someone was bobbing in and out of the water, struggling to find air. Behind was Casra, paddling furiously. _What has Linyu done?_ He headed toward hae, but before he could make it, hae head was no longer on the surface. He took a deep breath and dove under, accepting the sting of salt in his eyes. He kept gaining depth until he saw the faint glow of two blue eyes upon the sea floor. The distance was too long at this point, so he surfaced to catch his breath, inhaled as much as he possibly could, and dove again. Five strokes, ten strokes, twenty. The pain in his lungs pulled him back to the surface, but still he pushed further into the deep. Thirty, and finally he saw those blue glowing eyes. Turning back was not an option now – not without Linyu. Forty. Colorful flashes of light began to fill his vision and his muscles started to spasm and lock up - first his legs, then his arms, and then he started to panic. A hand grabbed his shirt, tugged to pull him out. There was nothing he could do but watch as those blue eyes so full of hope recede into the deep as flashes of light blotted out reality.
```

Comment:
Too lengthy of a paragraph of high adrenaline. The last few sentences are important, don't merge it with this long passage. Readers might miss it.